docs/pom.xml

```diff
@@ -310,12 +323,12 @@
                         </goals>
                         <phase>package</phase>
                         <configuration combine.children="append">
-                            <includeDependencySources>true</includeDependencySources>
-                            <includeTransitiveDependencySources>true</includeTransitiveDependencySources>
-                            <dependencySourceIncludes>
-                                <dependencySourceInclude>gov.nsa.datawave:*</dependencySourceInclude>
-                                <dependencySourceInclude>gov.nsa.datawave.webservices:*</dependencySourceInclude>
-                            </dependencySourceIncludes>
+                            <javadocVersion>1.8.0</javadocVersion>
```

Comment:
The version is defined in the parent, and that seems like something we should keep.